src/platform/macos.rs

```diff
@@ -301,6 +306,43 @@ pub fn get_cursor_pos() -> Option<(i32, i32)> {
     */
 }
 
+pub fn get_focused_window_id() -> Option<i32> {
```

Comment:
On my Mac, the window is the one named "StatusIndicator," and the "Follow remote window" feature doesn't work. This issue may not occur on every Mac.